docs/antora.yml

```diff
@@ -0,0 +1,20 @@
+name: maven-plugin
+title: Asciidoctor Maven Plugin
+version: '2.1'
+asciidoc:
+  attributes:
+    release-version: 2.1.0
+    maven-site-plugin-version: 3.9.0
+    project-repo: asciidoctor/asciidoctor-maven-plugin
+    uri-repo: https://github.com/{project-repo}
+    uri-asciidoc: http://asciidoc.org
+    uri-asciidoctor: http://asciidoctor.org
+    uri-asciidoctorj: https://github.com/asciidoctor/asciidoctorj
+    uri-examples: https://github.com/asciidoctor/asciidoctor-maven-examples
+    uri-maven: http://maven.apache.org
+    idprefix: ''
+    idseparator: '-'
```

Comment:
The idprefix and idseparator should only be set in the playbook, not the component descriptor.

Reply:
What should be the criteria? In practice it makes no diference right?

Reply:
This would be part of the style guide. It will indicate which attributes should be managed globally. All others can be managed by the component version or page.